src/templates/rsbuild/README.md

```diff
@@ -0,0 +1,35 @@
+# Create Deriv V2
+
+This template was generated using `create-deriv-v2`
+
+## Getting started
+
+To run the development server:
+
+```bash
+npm run dev
+```
+
+To generate a build:
+
+```bash
+npm run build
+```
+
+## Deploying to Cloudflare Pages
+
+In order to generate a deployment to Cloudflare Pages, ensure that the following secrets are set in the Github Actions:
+
+```bash
+CLOUDFLARE_ACCOUNT_ID=****
+CLOUDFLARE_API_TOKEN=****
```

Comment:
```suggestion
CLOUDFLARE_TOKEN=****
```

Reply:
My bad, Cloudflare uses `CLOUDFLARE_API_TOKEN` instead of `CLOUDFLARE_TOKEN`
<img width="751" alt="Screenshot 2024-03-15 at 2 27 52 PM" src="https://github.com/deriv-com/create-deriv-app/assets/103016120/95209f9d-2d9d-4ff2-9ec7-952cc03faef9">